corehq/tabs/tabclasses.py

```diff
@@ -596,7 +596,7 @@ def dropdown_items(self):
 
 
 class ApplicationsTab(UITab):
-    view = "corehq.apps.app_manager.views.view_app"
+    view = "default_app"
```

Comment:
If you check out ```app_manager/urls.py``` there are many different urls that all point to the same view function ```corehq.apps.app_manager.views.view_app```.  ```default_app``` refers to ```/a/<domain>/apps/```.  This seems like an anti-pattern, but fixing that is out of scope for this set of changes.